webapp/src/components/EditEstatePage/EditEstatePage.js

```diff
@@ -33,12 +32,16 @@ export default class EditEstatePage extends React.PureComponent {
     if (!id) {
       return (
         <Parcel x={x} y={y}>
-          {(_, isOwner, wallet) =>
-            this.renderEditEstate(getInitialEstate(x, y), isOwner, wallet)
+          {(parcel, isOwner, wallet) =>
```

Comment:
why are we changing the `_` to parcel here?

Reply:
my bad, was going for a refactor that didn't happen